sorc/ncep_post.fd/GFSPOST.F

```diff
@@ -1,57 +1,50 @@
 !> @file
-!
-!> Subprogram: pvetc      Compute potential vorticity, etc
-!!   Prgmmr: Iredell      Org: np23        Date: 1999-10-18
-!!
-!! Abstract: This subprogram computes
-!!             the Montgomery streamfunction
-!!               hm=cp*t+g*z
-!!             the specific entropy
-!!               s=cp*log(t/t0)-r*log(p/p0)
-!!             the Brunt-Vaisala frequency squared
-!!               bvf2=g/cp*ds/dz
-!!             the potential vorticity defined as
-!!               pvn=(av*ds/dz-dv/dz*ds/dx+du/dz*ds/dy)/rho/cp
-!!             the potential temperature
-!!               theta=t0*exp(s/cp)
-!!             the static stability
-!!               sigma=t/g*bvf2
-!!             and the potential vorticity in PV units
-!!               pvu=10**-6*theta*pvn
-!!
-!! Program history log:
-!!   1999-10-18  Mark Iredell
-!!
-!! Usage:  call pvetc(km,p,px,py,t,tx,ty,h,u,v,av,s,bvf2,pvn,theta,sigma,pvu)
-!!   Input argument list:
-!!     km       integer number of levels
-!!     p        real (km) pressure (Pa)
-!!     px       real (km) pressure x-gradient (Pa/m)
-!!     py       real (km) pressure y-gradient (Pa/m)
-!!     t        real (km) (virtual) temperature (K)
-!!     tx       real (km) (virtual) temperature x-gradient (K/m)
-!!     ty       real (km) (virtual) temperature y-gradient (K/m)
-!!     h        real (km) height (m)
-!!     u        real (km) x-component wind (m/s)
-!!     v        real (km) y-component wind (m/s)
-!!     av       real (km) absolute vorticity (1/s)
-!!   Output argument list:
-!!     hm       real (km) Montgomery streamfunction (m**2/s**2)
-!!     s        real (km) specific entropy (J/K/kg)
-!!     bvf2     real (km) Brunt-Vaisala frequency squared (1/s**2)
-!!     pvn      real (km) potential vorticity (m**2/kg/s)
-!!     theta    real (km) (virtual) potential temperature (K)
-!!     sigma    real (km) static stability (K/m)
-!!     pvu      real (km) potential vorticity (10**-6*K*m**2/kg/s)
-!!
-!! Modules used:
-!!   physcons       Physical constants
-!!
-!! Attributes:
-!!   Language: Fortran 90
-!!
-!!
-      subroutine pvetc(km,p,px,py,t,tx,ty,h,u,v,av,hm,s,bvf2,pvn,theta,sigma,pvu)
+!> @brief pvetc computes potential vorticity, etc.
+!>
+!><pre>
+!> This subprogram computes
+!>             the Montgomery streamfunction
+!>               hm=cp*t+g*z
+!>             the specific entropy
+!>               s=cp*log(t/t0)-r*log(p/p0)
+!>             the Brunt-Vaisala frequency squared
+!>               bvf2=g/cp*ds/dz
+!>             the potential vorticity defined as
+!>               pvn=(av*ds/dz-dv/dz*ds/dx+du/dz*ds/dy)/rho/cp
+!>             the potential temperature
+!>               theta=t0*exp(s/cp)
+!>             the static stability
+!>               sigma=t/g*bvf2
+!>             and the potential vorticity in PV units
+!>               pvu=10**-6*theta*pvn
```

Comment:
Doxygen will not preserve any formatting done with white-space. So this will all come out as one paragraph, without the line breaks you expect.

I would suggest you convert to a table like this:

```
computation | equation
-----------------|------------
Montgomery streamfunction | hm=cp*t+g*z
```

This will give you a nice table.

Reply:
Done. Thank you.